docker-compose.dev.yml

```
@@ -29,6 +29,14 @@ services:
     volumes:
       - elasticsearch-data:/usr/share/elasticsearch/data
 
+  dejavu:
+    image: appbaseio/dejavu:3.6.0
+    container_name: dejavu
+    ports:
+        - '1358:1358'
+    links:
+        - elasticsearch
```

Is this required? Services can connect to other service names using the service name (i.e. `elasticsearch` in this case) by default. We use `depends_on` for other service to specify service dependencies and control startup order.


```suggestion
    depends_on:
        - elasticsearch
```